workers/paymaster/src/index.ts

```diff
@@ -169,19 +174,48 @@ export default class PaymasterWorker extends Worker {
     );
 
     /**
-     * If today is not payday for workspace do nothing
+     * Define readable values
+     */
+    const isTimeToPay = PaymasterWorker.isTimeToPay(workspace.lastChargeDate);
+    const daysLeft = PaymasterWorker.daysAfterPayday(workspace.lastChargeDate) * -1;
+    const isFreePlan = currentPlan.monthlyCharge === 0;
+
+    /**
+     * Today is not payday for workspace
      */
-    if (!PaymasterWorker.isTimeToPay(workspace.lastChargeDate)) {
+    if (!isTimeToPay && !isFreePlan) {
+      /**
+       * If payday is coming then notify admins
+       *
+       * @todo do not notify if card is linked?
+       */
+      if (daysLeft <= DAYS_LEFT_THRESHOLD) {
+        /**
+         * Add task for Sender worker
+         */
+        await this.addTask(WorkerNames.EMAIL, {
```

Comment:
paymaster starts every N hours, right? So a user would get several similar emails per day? I think it should be restricted to one email per day.

Reply:
yep, it would be great, but we need a research for the best implementation